gcovr/workers.py

```diff
@@ -110,10 +129,19 @@ def wait(self):
         for w in self.workers:
             self.add(None)
         for w in self.workers:
-            w.join()
+            while w.is_alive():
+                w.join(timeout=0.1)
```

Comment:
Why is it necessary to use a timeout here?

Reply:
I found the answer here (https://stackoverflow.com/questions/3788208/threading-ignores-keyboardinterrupt-exception) - it was the only way I could get a KeyboardInterrupt to get reported during a join. I don't like it myself, but I think its better than making the threads daemons.

Reply:
I've added a comment and gone with timeout=1, for my sparc box that means things stop within 10 seconds (the gcov run is ~8 or 9s).